autoload/go/cmd.vim

```diff
@@ -124,6 +135,35 @@ function! go#cmd#Test(compile, ...)
     endif
 endfunction
 
+function! go#cmd#TestFocused(...)
+    let test = search("func Test", "bcsW")
```

Comment:
Can you change the flags to `bsnW` ? Don't move the cursor. Also I would be happy if you could add the flag meanings as comment so someone else reading the code can have some idea :)